docs/example/AKAP9_BRAF.html

```diff
@@ -1239,6 +1239,9 @@ <h1 class="h2">AKAP9--BRAF</h1>
                 </li>
                                 <li class="nav-item">
                     <a class="nav-link" onclick="toggleCard(this, 'card-starfusion')">starfusion</a>
+                </li>
+                                <li class="nav-item">
+                    <a class="nav-link" onclick="toggleCard(this, 'card-starfusion_lr')">starfusion_lr</a>
```

Comment:
I don't undestand why is this called starfusion_lr and not ctat-lr-fusion

Reply:
Yes I agree